kurtosis/main.star

```diff
@@ -0,0 +1,72 @@
+KURTOSIS_PACKAGE_INDEXER_IMAGE = "kurtosistech/kurtosis-package-indexer"
+KURTOSIS_PACKAGE_INDEXER_DEFAULT_VERSION = "0.0.6"
+KURTOSIS_PACKAGE_INDEXER_PORT_NUM = 9770
+
+AWS_S3_BUCKET_SUBFOLDER = "kurtosis-package-indexer"
+
+
+def run(
+    plan,
+    github_user_token="",                        # type: string
+    aws_access_key_id="",                        # type: string
+    aws_secret_access_key="",                    # type: string
+    aws_bucket_region="",                        # type: string
+    aws_bucket_name="",                          # type: string
+    aws_bucket_user_folder="",                   # type: string
+    kurtosis_package_indexer_custom_version="",  # type: string
```

Comment:
@gbouv do we still use this syntax, with the new docstrings that are rolling out?

Reply:
Ah, good point, we're not. This PR was initially subnmitted before we changed it. I'll update this 👍

Reply:
All good otherwise